docs/src/app/(public)/(content)/react/components/alert-dialog/page.mdx

```diff
@@ -87,3 +87,91 @@ To implement this, both dialogs should be controlled. The confirmation dialog ma
 Use the `[data-nested-dialog-open]` selector and the `var(--nested-dialogs)` CSS variable to customize the styling of the parent dialog. Backdrops of the child dialogs won't be rendered so that you can present the parent dialog in a clean way behind the one on top of it.
 
 <Demo compact path="../dialog/demos/close-confirmation" />
+
+### Detached triggers
+
+An alert dialog can be controlled by a trigger located either inside or outside the `<AlertDialog.Root>` component.
+For simple, one-off interactions, place the `<AlertDialog.Trigger>` inside `<AlertDialog.Root>`, as shown in the example at the top of this page.
+
+However, if defining the alert dialog's content next to its trigger is not practical, you can use a detached trigger.
+This involves placing the `<AlertDialog.Trigger>` outside of `<AlertDialog.Root>` and linking them with a `handle` created by the `AlertDialog.createHandle()` function.
+
+```jsx title="Detached triggers"
+const demoAlertDialog = AlertDialog.createHandle();
+
+<AlertDialog.Trigger handle={demoAlertDialog}>Open</AlertDialog.Trigger>
+
+<AlertDialog.Root handle={demoAlertDialog}>
+  ...
+</AlertDialog.Root>
+```
```

Comment:
Using character syntax highlighting (`"demoAlertDialog"` in the code block fence frontmatter) would be helpful